PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/DynamicThumbstick.rbxmx

```diff
@@ -58,9 +58,23 @@ local FadeInAndOutMaxAlpha = 0.35
 local FadeInAndOutHalfDuration = 0.3
 
 --[[ Local functionality ]]--
-	
+
+local function isDynamicThumbstickEnabled()
+	return ThumbstickFrame and ThumbstickFrame.Visible
+end
+
+local function enableAutoJump(humanoid)
+	if humanoid and isDynamicThumbstickEnabled() then
+		local shouldRevert = humanoid.AutoJumpEnabled == false
```

Comment:
are we storing shouldRevert here because the logic is complicated to store in RevertAutoJumpEnabledToFalse?